ui/src/app/applications/components/application-details/application-resource-list.tsx

```diff
@@ -102,9 +102,15 @@ export const ApplicationResourceList = (props: ApplicationResourceListProps) =>
                                             </Consumer>
                                         )}
                                     </div>
-                                    <div className='columns small-1 xxxlarge-1'>{[res.group, res.kind].filter(item => !!item).join('/')}</div>
-                                    <div className='columns small-1 xxxlarge-1'>{res.syncWave || '-'}</div>
-                                    <div className='columns small-2 xxxlarge-1'>{res.namespace}</div>
+                                    <div className='columns small-1 xxxlarge-1' title={[res.group, res.kind].filter(item => !!item).join('/')}>
```

Comment:
Would you consider using the tooltip component from argo-ui, which is already available, instead of the native tooltip?

Reply:
Thanks for your review! I applied it as a tooltip in argo-ui as you said, and it looked much prettier :)
https://github.com/argoproj/argo-cd/commit/df59d795364acff62460f67a65c17b62df7d2fe4